src/_P020_Ser2Net.ino

```diff
@@ -102,9 +105,14 @@ boolean Plugin_020(uint8_t function, struct EventStruct *event, String& string)
         options[2] = F("RFLink");
         addFormSelector(F("Event processing"), F("p020_events"), 3, options, NULL, choice);
       }
-      addFormNumericBox(F("RX Receive Timeout (mSec)"), F("p020_rxwait"), P020_RX_WAIT, 0);
+      addFormNumericBox(F("RX Receive Timeout (mSec)"), F("p020_rxwait"), P020_RX_WAIT, 0, 20);
+      addFormPinSelect(F("Reset target after init"), F("p020_resetpin"), P020_RESET_TARGET_PIN);
+
+      addFormNumericBox(F("RX extra long messages (bytes)"), F("p020_rx_buffer"), P020_RX_BUFFER, 256, 1024);
```

Comment:
The label should reflect what the value is used for, not when to use it.
The note you added is clear enough.
So the label should be as simple as `"RX Buffer Size"`

Reply:
Hmm it doesn't look like this buffer size is actually used????

Reply:
better label implemented, thanks for the hint

Reply:
done